install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+In OpenShift a Persistent Volume Claim (PVC) will bind to a Persistent Volume (PV) based on matching the
```

Comment:
We've been adding a line to dissuade "1)" to the storage provider docs recently, e.g. in https://docs.openshift.com/enterprise/latest/install_config/persistent_storage/persistent_storage_nfs.html:

"While the NFS-specific information contained in a PV definition could also be defined directly in a pod definition, doing so does not create the volume as a distinct cluster resource, making the volume more susceptible to conflicts."